### PR TITLE
mce version numbers need fixed https://issues.redhat.com/browse/ACM-8416

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/cluster_lifecycle/release_image_connected.adoc [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 You might want to use the same release image for all of your clusters. To simplify, you can create your own custom list of release images that are available when creating a cluster. Complete the following steps to manage your available release images:
 
-. Fork the link:https://github.com/stolostron/acm-hive-openshift-releases/tree/backplane-2.3[acm-hive-openshift-releases GitHub repository 2.7 branch].
+. Fork the link:https://github.com/stolostron/acm-hive-openshift-releases/tree/backplane-2.x[acm-hive-openshift-releases GitHub repository 2.7 branch].
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/release_image_connected.adoc]: Just a heads up, the  _acm-hive-openshift-releases GitHub repository 2.7 branch_ link is broken
Screenshot 2023-11-13 at 3 35 15 PM

Comment (dockerymick) [reply clusters/cluster_lifecycle/release_image_connected.adoc]: link:https://github.com/stolostron/acm-hive-openshift-releases
```suggestion
. Fork the link:https://github.com/stolostron/acm-hive-openshift-releases/tree/backplane-2.x[acm-hive-openshift-releases GitHub repository 2.7 branch].
```

**Review by dockerymick — APPROVED** [review]
Thanks for making the changes. There was a broken link. Otherwise, this looks good

File: clusters/cluster_lifecycle/release_image_connected.adoc (hunk 1 above)
Comment (swopebe) [reply clusters/cluster_lifecycle/release_image_connected.adoc]: good catch on this link, however, I am afraid asking user to fork a branch, but sending to a repo link is not gonna work, so now I have to find the correct one.

Comment (swopebe) [reply clusters/cluster_lifecycle/release_image_connected.adoc]: That link doesn't even have 2.9 branch ugh.

Comment (swopebe) [reply clusters/cluster_lifecycle/release_image_connected.adoc]: Ok, I got it! Who knows if it is what it should be but I assume it goes to MCE 2.4. Really scary that it was previously pointing to 2.7 but I have a bug from that for 2.8 and earlier anyway. Looks like it was not correct to begin with. @dockerymick can you review again?

**Pull request merged**